.travis.yml

```diff
@@ -9,5 +9,6 @@ install:
   - pip install coveralls
 script:
   - py.test
+  - mypy result/result.py
```

Comment:
Possibly try `python3 -m mypy result/result.py`. Don't know if python 3 exists in the environment, but if so that should work.